src/main/kotlin/de/lobbenmeier/stefan/settings/ui/SettingsUI.kt

```diff
@@ -59,6 +61,36 @@ fun SettingsUI(settings: Settings, save: (Settings) -> Unit, cancel: () -> Unit)
             modifier = Modifier.weight(1f).verticalScroll(rememberScrollState()),
             verticalArrangement = Arrangement.spacedBy(32.dp),
         ) {
+            Section("Binaries") {
+                FixedChoiceInput(
+                    "Yt-Dlp Source",
+                    mutableSettings.ytDlpSource,
+                    onValueChange = { mutableSettings = mutableSettings.copy(ytDlpSource = it) },
+                    options = YtDlpLocation.entries
+                )
```

Comment:
There still is some bug here that saving this setting does not trigger a download, not sure why. Currently it requires a restart before it works. Will check some other day, then this can be merged 😄

Reply:
Looks like this is fixed now